pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -34,11 +34,11 @@ import (
 	"k8s.io/apiserver/pkg/registry/rest"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	serverstorage "k8s.io/apiserver/pkg/server/storage"
+	"k8s.io/client-go/util/retry"
```

Comment:
`retry` depends on `k8s.io/apimachinery`, see: https://github.com/mengqiy/kubernetes/blob/05deb21f5c84324a24660f8e85e8e5d4f21a7419/staging/src/k8s.io/client-go/util/retry/util.go#L22-L23.

IMHO, it's not a generic util. Not sure where it should live.